File-Formats/PDF/Working-with-OCR/Docker.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Performing OCR in Docker | Syncfusion
```

Comment:
Perform OCR on PDF and image files in Docker

Reply:
Changed the content